docs/faq.md

```
@@ -0,0 +1,54 @@
+
+# Frequently Asked Questions (FAQ)
+
+This FAQ summarizes common issues encountered by ChatALL users and provides suggested solutions based on discussions in the GitHub Issues section.
+
+If you still haven't found the answer you're looking for, please submit an [Issue](https://github.com/ai-shifu/ChatALL/issues/new/choose).
+
+---
+
+### [#245](https://github.com/ai-shifu/ChatALL/issues/245), [#196](https://github.com/ai-shifu/ChatALL/issues/196)
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)